src/gui/update.rs

```diff
@@ -145,6 +145,8 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
                     // Update the addon with the newly parsed information.
                     addon.update_addon(updated_addon);
                 }
+
+                ajour.addons.sort();
```

Comment:
Remove this if you want to keep addons that were just updated at the top of the list until 'Refresh' is explicitly clicked.